src/main/java/info/blockchain/wallet/api/data/Status.java

```diff
@@ -19,20 +19,9 @@
     isGetterVisibility = Visibility.NONE)
 public class Status {
 
-    @JsonProperty("code")
```

Comment:
I changed the caller methods to return `Response<Status>`, which supports parsing the response code and wraps the status object. For some reason, returning `Status` on it's own would always return `code = 0`, so I let Retrofit do it for me.